sim_benchmark/robot_df/turtlebot/model.urdf

```diff
@@ -0,0 +1,750 @@
+<?xml version="1.0" ?>
+<!-- =================================================================================== -->
+<!-- |    This document was autogenerated by xacro from robots/kobuki_hexagons_kinect.urdf.xacro | -->
+<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
+<!-- =================================================================================== -->
+<!--
+    - Base      : kobuki
+    - Stacks    : hexagons
+    - 3d Sensor : kinect
+-->
+<robot name="turtlebot" xmlns:controller="http://playerstage.sourceforge.net/gazebo/xmlschema/#controller" xmlns:interface="http://playerstage.sourceforge.net/gazebo/xmlschema/#interface" xmlns:sensor="http://playerstage.sourceforge.net/gazebo/xmlschema/#sensor" xmlns:xacro="http://ros.org/wiki/xacro">
+
+  <link name="base_link">
+    <visual>
+      <geometry>
+        <!-- new mesh -->
+        <mesh filename="/home/ubuntu/Srcs/SocialRobot/simu_benchmark/robot_df/turtlebot/turtlebot/kobuki_description/meshes/main_body.dae"/>
```

Comment:
do not use hardcoded path.

Reply:
Pybullet can work with relative path, but pygazebo send an error when loading if relative path is used. I'll find a way.

Reply:
Fixed.